### PR TITLE
fix macro config input field width size [SDNTB-876]

### PR DESCRIPTION
**Pull request by devketanpro**
image

### CODE REVIEW

File: scripts/apps/users/views/user-preferences.html [hunk 1]
```diff
@@ -423,15 +423,22 @@ <h4 class="sd-heading sd-text-align--left sd-text--sans sd-heading--h4 sd-margin
...
                 </li>
 
-                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start">
-                    <h3 id="macros-config" class="sd-heading sd-text-align--left sd-text--sans sd-heading--h3" translate>{{preferences['macro_config'].label}}</h3>
-                    <div class="sd-container sd-container--flex sd-container--gap-none sd-container--direction-column sd-radius--medium sd-panel-bg--000 sd-shadow--z2 sd-padding--3 sd-state--focus sd-margin-b--1">
-                        <div sd-info-item>
-                            <div ng-repeat="(key, field) in preferences.macro_config.fields">
-                            <label class="form-label" translate>{{ key }}</label>
-                            <input type="text" ng-model="preferences['macro_config'].fields[key]" />
+                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start"></li>
```
Comment (tomaskikutis) [inline scripts/apps/users/views/user-preferences.html]: what's the purpose to have an empty `<li>` tag?

**Comment by tomaskikutis** [comment]
since this is a new functionality, can you add a screenshot of how it looks?

File: scripts/apps/users/views/user-preferences.html (hunk 1 above)
Comment (devketanpro) [reply scripts/apps/users/views/user-preferences.html]: @fritzSF suggested these changes, and I’ve applied them as is.

Comment (fritzSF) [reply scripts/apps/users/views/user-preferences.html]: I don't think the `</li>` closing tag is there on purpose. Actually it is an error in my example code, most likely that VS Code added it. Sorry @devketanpro, please remove the closing tag there,

File: scripts/apps/users/views/user-preferences.html [hunk 2]
```diff
@@ -423,7 +423,7 @@ <h4 class="sd-heading sd-text-align--left sd-text--sans sd-heading--h4 sd-margin
                     </div>
                 </li>
 
-                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start"></li>
+                <li ng-show = "preferences['macro_config']" class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start">
```
Comment (tomaskikutis) [inline scripts/apps/users/views/user-preferences.html:426]: do the last 2 empty `<li>` tags in this file server any purpose? If not, drop them

**Review by fritzSF — APPROVED** [review]

File: scripts/apps/users/views/user-preferences.html (hunk 2 above)
Comment (devketanpro) [reply scripts/apps/users/views/user-preferences.html:426]: done

**Pull request merged**